src/pages/ReportDetailsPage.js

```diff
@@ -132,11 +132,9 @@ class ReportDetailsPage extends Component {
                             style={styles.reportDetailsTitleContainer}
                         >
                             <Avatar
-                                isChatRoom={isChatRoom}
-                                isArchivedRoom={ReportUtils.isArchivedRoom(this.props.report)}
                                 containerStyles={[styles.singleAvatarLarge, styles.mb4]}
                                 imageStyles={[styles.singleAvatarLarge]}
-                                source={this.props.report.icons[0]}
+                                source={_.first(OptionsListUtils.getAvatarSources(this.props.report))}
```

Comment:
I wanna note that this breaks the icons here, but I've already fixed that in my PR with this commit (I used your updates to RoomHeaderAvatars too!): https://github.com/Expensify/App/pull/7957/commits/50300783f38c2b6d5a3397b4925ec070751e1305

It's not really relevant in this PR, so no need to add it here, I just noticed it when I added reportDetailsView for the policyExpenseChats (which is something I have to do for my PR). The only reason I bring it up is that if anyone notices this is off, it's already fixed!